### PR TITLE
docs: explain the enforcer_options

### PR DESCRIPTION
**Pull request by hectorj2f**
## Type of change

Add new documentation
### What should this PR do?

related to: https://github.com/chainguard-dev/edu/issues/495
Add documentation on how to use the enforcer options.

### Why are we making this change?

We added few features to Enforce so users could configure the behavior of their tenant clusters by passing the flag `--opt` to the chainctl cluster installation.

### CODE REVIEW

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/configure-enforcer-options-during-installation.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+For this example on a GKE, EKS, or AKS cloud infrastructure, we enable our cluster webhook to fail open by using the next command. 
+
+```sh
+chainctl cluster install --group=$GROUP_ID --context $CLUSTER --opt=webhook_fail_open=true
+```
+
+In this other example using a _private_ or on-prem cluster, we configure our Chainguard cluster to enable the cluster image policy cache and set `opt-out` as enforcement mode.
+
+```sh
+chainctl cluster install --group=$GROUP_ID --private --context $CLUSTER --opt=webhook_fail_open=true --opt=namespace_enforcement_mode=opt-out
```
Comment (hectorj2f) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/configure-enforcer-options-during-installation.md:55]: I am a newbie here, should we add a paragraph to explain how to verify whether the changes had the desired effect ?

**Comment by hectorj2f** [comment]
I added a DNM (do-not-merge) waiting for https://github.com/chainguard-dev/customer-issues/issues/105 so we can add support for the three enforcer options. But I am happy to hear any feedback so I can address all the comments and merge it whenever all these options landed.

**Review by ltagliaferri — APPROVED** [review]
@hectorj2f I made some small edits, one being the title as we have another that is called "Installation". Please eyeball again and let me know if you would like any additional changes or anything reverted back. Thank you for all the docs!

**Comment by hectorj2f** [comment]
@ltagliaferri Looks good to me 🙏🏻 ! Thanks.

**Pull request merged**